core/txpool/blobpool/evictheap.go

```diff
@@ -49,20 +50,17 @@ type evictHeap struct {
 func newPriceHeap(basefee *uint256.Int, blobfee *uint256.Int, index *map[common.Address][]*blobTxMeta) *evictHeap {
 	heap := &evictHeap{
 		metas: index,
-		index: make(map[common.Address]int),
+		index: make(map[common.Address]int, len(*index)),
 	}
 	// Populate the heap in account sort order. Not really needed in practice,
 	// but it makes the heap initialization deterministic and less annoying to
 	// test in unit tests.
-	addrs := make([]common.Address, 0, len(*index))
-	for addr := range *index {
-		addrs = append(addrs, addr)
-	}
+	addrs := maps.Keys(*index)
 	sort.Slice(addrs, func(i, j int) bool { return bytes.Compare(addrs[i][:], addrs[j][:]) < 0 })
```

Comment:
you can also change it to `slices.SortFunc(addrs, common.Address.Cmp)`

Reply:
Great idea.